src/wasm-interpreter.h

```diff
@@ -1130,8 +1133,20 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
   }
   Flow visitDrop(Drop* curr) {
     NOTE_ENTER("Drop");
+    seenUnknownLocalGet = false;
     Flow value = visit(curr->value);
     if (value.breaking()) {
+      // Handle the case where all we don't know to perform the drop is a local
+      // value, which we do not need to know if there are no other side-effects.
+      // Doesn't apply to globals since these would error when module is given.
```

Comment:
Could you elaborate on what this means? Why do globals cause error when module is given?

Reply:
Good catch. Looked at this again and my assumption there is indeed wrong. I believe I based this on `visitGlobalGet` using `Module::getGlobal`, which asserts that the global exists, but I missed that the global may exist but not have a known value. Going to fix :)